### PR TITLE
Added more explicit instructions to locally deploy/test mungegithub

### PR DESCRIPTION
**Pull request by foxish**
- Added some instructions that I felt were missing based on my experience running it locally.
- Reworded some of the build instructions.

### CODE REVIEW

**Comment by foxish** [comment]
@apelisse @eparis PTAL and let me know if this is any good. I felt like some instructions could be expanded on further in the README. Please let me know if there are any ambiguities in it.

File: mungegithub/submit-queue/pv-local.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  name: submit-queue-cache
+  labels:
+    type: local
+spec:
+  capacity:
+    storage: 10Gi
+  accessModes:
+    - ReadWriteOnce
+  hostPath:
+    path: "/tmp"
```
Comment (apelisse) [inline mungegithub/submit-queue/pv-local.yaml]: Can you add the missing new line at the end?

**Comment by apelisse** [comment]
Beside the small nit, LGTM

File: mungegithub/submit-queue/pv-local.yaml (hunk 1 above)
Comment (foxish) [inline mungegithub/submit-queue/pv-local.yaml]: Done.

**Comment by foxish** [comment]
@lavalamp Updated the README with instructions to deploy using configmaps/repository-specific deployment files. PTAL.

File: mungegithub/README.md [hunk 2]
```diff
@@ -12,23 +12,37 @@ One may also look in the `example-one-off` directory for a small skeleton progra
...
 ```
 
-After this has successfully deployed to the test cluster in read-only mode running in production required running any required `kubectl config` commands and then running
+It can finally deployed as:
 ```sh
-REPO=docker.io/eparis APP=submit-queue KUBECONFIG=/path/to/kubeconfig READONLY=false make deploy
+TARGET=kubernetes REPO=docker.io/$USERNAME APP=submit-queue KUBECONFIG=/path/to/kubeconfig make deploy
 ```
 
+After this has successfully deployed to the test cluster in read-only mode, running in product ion involves running any required `kubectl config` commands to point to the production cluster, pushing a configmap if necessary, and then running:
```
Comment (lavalamp) [inline mungegithub/README.md]: s/product ion/production/

**Comment by lavalamp** [comment]
just one nit, thanks

File: mungegithub/README.md (hunk 2 above)
Comment (foxish) [inline mungegithub/README.md]: Done.

**Comment by lavalamp** [comment]
lgtm

**Pull request merged**